nats-base-client/jsmstream_api.ts

```diff
@@ -83,6 +109,9 @@ export class StreamAPIImpl extends BaseApiClient implements StreamAPI {
     validateStreamName(name);
     const old = await this.info(name);
     const update = Object.assign(old.config, cfg);
+    update.mirror = convertStreamSourceDomain(update.mirror);
```

Comment:
Not sure updating a stream and changing the mirror/source actually works (in that I don't think the server supports that), so maybe the reason why it is only on "AddStream" in the go client?

Reply:
it should be a noop regardless - because it has to have "domain" set which won't be, so it is ignored. I just wanted to trap it for uniformity in case it updates later.